Traceback (most recent call last):
  File "/gpfs/ysm/project/girgenti/sss253/conda_envs/scEpilock/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/gpfs/ysm/project/girgenti/sss253/conda_envs/scEpilock/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "binary_classifier/__main__.py", line 48, in <module>
    y_true, y_pred = tester.test()
  File "binary_classifier/train/tester.py", line 29, in test
    model = self.model(self.n_class).to(device)
  File "binary_classifier/model/model.py", line 73, in __init__
    super(scEpiLock, self).__init__()
TypeError: super(type, obj): obj must be an instance or subtype of type
positive size =  (399457,)
negative size =  (711878,)
label size =  (399457, 7)
-----------train size concate------------
(323559,)
(323559, 7)
-----------eval size concate------------
(35952,)
(35952, 7)
-----------test size concate------------
(39946,)
(39946, 7)
-----------------shape before add RC -------------
(39946,)
(39946, 7)
train start time:  2022-04-21 20:56:13.721131
train end time:  2022-04-21 20:56:13.721154
